bin/_docker.sh

```diff
@@ -46,7 +46,7 @@ docker_build() {
     file=$1
     shift
 
-    rootdir=$( cd "$bindir"/.. && pwd )
+    rootdir=${ROOTDIR:-$( cd "$bindir"/.. && pwd )}
```

Comment:
needed in order to override the root dirrectory for docker-build-policy-controller